Re-add working with incubating codebases (#1170)
publiccodenet/about: activities/codebase-stewardship/incubating-codebases.md

---
type: guide
explains: important steps in working with a codebase in incubation
---

# Working with a codebase in incubation

> At least once each year the codebase stewards organizes a meeting with the codebase community to do a forward look.

Reviewer: ```suggestion
At least once each year the codebase stewards organize a meeting with the codebase community to do a forward look.
```

> The meeting should cover what the ambitions of the community related to meeting more requirements in the Standard for Public Code will be the next year.

Reviewer: ```suggestion
The meeting should cover the community's ambitions in meeting more Standard for Public Code requirements in the coming year.
```

At the latest, a forward look like this should happen about three months before the [general assembly](../../organization/governance-model.md#general-assembly).
It should be clearly communicated that the information collected in the forward look will feed into a report for the general assembly.
The forward look could either be part of a regular product steering meeting if such exist, or be a separate meeting.

> Afterwards, the codebase stewards prepares a "state of the codebases", covering all codebases in stewardship, to the general assembly so that the members can make informed decisions during it.

Reviewer: ```suggestion
Afterwards, the codebase stewards prepare an overview of all codebases in stewardship for members so they can make informed decisions during the general assembly.
```

Of particular interest for the codebase stewards is if the general assembly wants to prioritize a codebase for strategic reasons, even if compliance with the Standard for Public Code is unlikely the coming year.
Such decision will give the codebase stewards the mandate to continue stewarding the incubated codebase.

> Alternatively, the members can decide that stop the incubation of a codebase that doesn't present any renewed ambitions and simply end the lifecycle.

Reviewer: ```suggestion
Alternatively, the members can decide to stop the incubation of a codebase that doesn't present any renewed ambitions and simply end stewardship.
```
